### ISSUE TITLE
tar: This does not look like a tar archive while building AppImage

### ISSUE THREAD

**Issue by fusion809** [OpenRA/d2#134]
Hi,

The AppImage for the latest d2 commit (68b08f28f4d9bae1f4f70005e97e2c74ba2e1e80) fails to build with the error:

```bash
Installing OpenRA engine to /data/GitHub/others/d2/packaging/linux/D2Mod.appdir/usr/lib/openra
Installing OpenRA common mod files to /data/GitHub/others/d2/packaging/linux/D2Mod.appdir/usr/lib/openra
Downloading dependencies
tar: This does not look like a tar archive
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Thanks for your time.

**Comment by evgeniysergeev** [comment]
Is it fixed by #108 or not?

**Comment by fusion809** [comment]
Sure is.
